colander/__init__.py

```diff
@@ -205,6 +205,15 @@ def __str__(self):
         result of an execution of this exception's ``asdict`` method"""
         return pprint.pformat(self.asdict())
 
+
+class ExtraItemsError(Invalid):
+    def __init__(self, node, extras, msg=None):
+        if msg is None:
+            msg = _('Unrecognized items')
```

Comment:
In this case it will be evaluated on module import time. Is it not too early? I think many program make message translation configuration on more later stage.